code/services-application/search-service/resources/static/search/serp.scss

```diff
@@ -228,6 +231,11 @@ header {
   }
 }
 
+#theme {
```

Comment:
I think one thing that might be good is to have `#theme` be `display:none` unless `body[data-has-js='true']`.  This means  the experience degrades gracefully if you disable JS.